run-tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+
+TESTING_BRANCH=${TESTING_BRANCH:-dev}
+KUBEADDONS_ENTERPRISE_PATH=${KUBEADDONS_ENTERPRISE_PATH:-kubeaddons-enterprise}
+KUBEADDONS_ENTERPRISE_TESTS_PATH="./tests"
+KUTTL_TEST_CONFIGURATION_PATH="kuttl-test.yaml"
+printf "testing kubeaddons-enterprise branch %s\n" "$TESTING_BRANCH"
+
+if [ -d "./addons" ]
+then
+  printf "Running inside the kubeaddons-enterprise repository"
+  KUBEADDONS_ENTERPRISE_PATH="."
+  KUBEADDONS_ENTERPRISE_TESTS_PATH="kubeaddons-enterprise-tests/tests/kubeaddons-enterprise"
+  KUTTL_TEST_CONFIGURATION_PATH="kubeaddons-enterprise-tests/kuttl-test.yaml"
+elif [ -d "$KUBEADDONS_ENTERPRISE_PATH" ]
+then
+  pushd $KUBEADDONS_ENTERPRISE_PATH
+  git checkout ${TESTING_BRANCH}
+  popd
+else
+  git clone --depth 1 https://github.com/mesosphere/kubeaddons-enterprise.git --branch ${TESTING_BRANCH} --single-branch
+fi
+
+KUBEADDONS_ENTERPRISE_ABS_PATH=$(realpath -L "${KUBEADDONS_ENTERPRISE_PATH}")
+
+for i in $(find "${KUBEADDONS_ENTERPRISE_PATH}"/addons -mindepth 2 -type d)
+do
+    printf "addon directory detected: %s\n" "$i"
+    if [ ! -d "${KUBEADDONS_ENTERPRISE_TESTS_PATH}/$i" ]
+    then
+      printf "missing tests for addon directory %s\n" "${KUBEADDONS_ENTERPRISE_TESTS_PATH}/$i"
+      exit 1
+    fi
+done
+
+go get github.com/jstemmer/go-junit-report
```

Comment:
Can you specify a version rather than get latest each time?

Reply:
added using go module, once https://github.com/kudobuilder/kuttl/pull/128 is released in a KUTTL version we will remove `go-junit-report` from this repository.